From 349c07de4c81ec1984f785f4b82c24a1356cff1c Mon Sep 17 00:00:00 2001
From: "Kevin F. Haggerty" <haggertk@lineageos.org>
Date: Tue, 19 Feb 2019 20:40:48 -0700
Subject: [PATCH 08/10] sepolicy: Label all the livedisplay service
 implementations

* There is still a pending samsung version, and who knows what comes
  next.  As opposed to making this list grow for each discrete
  version, get them all in one fell swoop using the power of regex.

Change-Id: Ia099d2e026082fd0a1341210d5e678b9c2801065

# Conflicts:
#	common/vendor/file_contexts
---
 common/vendor/file_contexts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/common/vendor/file_contexts b/common/vendor/file_contexts
index f7b8346..e954592 100644
--- a/common/vendor/file_contexts
+++ b/common/vendor/file_contexts
@@ -5,7 +5,7 @@
 /(vendor|system/vendor)/bin/hw/android\.hardware\.light@2\.0-service\.aw2013 u:object_r:hal_light_default_exec:s0
 
 # LiveDisplay HAL
-/(vendor|system/vendor)/bin/hw/vendor\.lineage\.livedisplay@2\.0-service-sysfs    u:object_r:hal_lineage_livedisplay_sysfs_exec:s0
+/(vendor|system|system/vendor)/bin/hw/(vendor\.)?lineage\.livedisplay@2\.0-service.* u:object_r:hal_lineage_livedisplay_sysfs_exec:s0
 
 # Trust HAL
 /(vendor|system/vendor)/bin/hw/vendor\.lineage\.trust@1\.0-service u:object_r:hal_lineage_trust_default_exec:s0
-- 
2.17.1

